### PR TITLE
tracing/python: update for v1.0

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
### What does this PR do?

Update the docs for the ddtrace 1.0 update. These features have either been deprecated or removed already.

### Motivation

ddtrace 1.0

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/troubleshooting/tracer_debug_logs.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ If there are errors that you don't understand, or if traces are reported as flus
 {{< /programming-lang >}}
 {{< programming-lang lang="python" >}}
 
-For more visibility, include `DD_LOGGING_RATE_LIMIT=0`.
+For more visibility, include `DD_TRACE_LOGGING_RATE=0`.
```
Comment (brettlangdon) [inline content/en/tracing/troubleshooting/tracer_debug_logs.md:223]: Is this needed anymore? at least when debug logging is enabled we disable the rate limit (iirc)?

Comment (Kyle-Verhoog) [reply content/en/tracing/troubleshooting/tracer_debug_logs.md:223]: Hmm I don't see any logic for this in the library. Only place that the rate limit appears to be set is through the environment variable.

File: content/fr/tracing/troubleshooting/tracer_debug_logs.md [hunk 2]
```diff
@@ -227,7 +227,7 @@ Si vous constatez des erreurs que vous ne comprenez pas, ou si les traces sont s
 {{< /programming-lang >}}
 {{< programming-lang lang="python" >}}
 
-Pour plus de visibilité, ajoutez `DD_LOGGING_RATE_LIMIT=0`.
+Pour plus de visibilité, ajoutez `DD_TRACE_LOGGING_RAT=0`.
```
Comment (majorgreys) [inline content/fr/tracing/troubleshooting/tracer_debug_logs.md:230]:
```suggestion
Pour plus de visibilité, ajoutez `DD_TRACE_LOGGING_RATE=0`.
```

**Review by cswatt — APPROVED** [review]

**Review by brettlangdon — APPROVED** [review]

**Comment by Kyle-Verhoog** [comment]
@cswatt can this be merged?

**Pull request merged**